app/scripts/controllers/citationInfo.ctrl.js

```diff
@@ -7,6 +7,38 @@ angular.module('yourStlCourts').controller('CitationInfoCtrl', function (faqData
   ctrl.paymentUrl = "";
   ctrl.citationCourtLocations = {};
 
+  ctrl.groupCitationsByDL = function(){
+    var dlNum = "";
+    var dlState = "";
+    var groupedCitations = new Array();
+
+    for(var citationIndex = 0; citationIndex < citations.length; citationIndex++){
+      var groupedCitation = citations[citationIndex];
+      dlNum = groupedCitation.drivers_license_number;
+      dlState = groupedCitation.drivers_license_state;
+      if (dlNum === ""){
+        //in the event the defendant does not have a DL Num assign one so the array has an index
+        dlNum = "A1";
```

Comment:
Why `A1` ? Is that picked at random? If we're using something as a placeholder, it should be clearer that it's a placeholder. And you'll want to define it in const pattern like `PLACEHOLDER_DL_NUM = 'NO_DL_NUM'` or something